src/components/Table.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import ComponentFactory from './ComponentFactory';
+import { getNestedValue } from '../utils/get-nested-value';
+import styled from '@emotion/styled';
+import { colorMap, size } from '../components/dev-hub/theme';
```

Comment:
Can we use shortened import here? 
```suggestion
import { getNestedValue } from '~utils/get-nested-value';
import styled from '@emotion/styled';
import { colorMap, size } from '~components/dev-hub/theme';
```